lmdeploy/serve/openai/api_server.py

```diff
@@ -852,54 +849,63 @@ def encode(prompt: str, do_preprocess: bool, add_bos: bool):
             length.append(len(ids))
         return EncodeResponse(input_ids=encoded, length=length)
 
+
 """ PD Disaggregation API Begin """
-@router.get("/distserve/engine_info")
+
+
+@router.get('/distserve/engine_info')
 async def engine_info():
     engine = VariableInterface.async_engine.engine
 
-    response = DistServeEngineConfig(
-        tp_size=engine.engine_config.tp,
-        dp_size=engine.engine_config.dp,
-        pp_size=None,
-        ep_size=engine.engine_config.ep,
-        dp_rank=engine.engine_config.dp_rank,
-        block_size=engine.engine_config.block_size,
-        num_cpu_blocks=engine.scheduler.block_manager.num_cpu_blocks,
-        num_gpu_blocks=engine.scheduler.block_manager.num_gpu_blocks,
-        available_nics=engine.engine_config.available_nics
-    )
+    response = DistServeEngineConfig(tp_size=engine.engine_config.tp,
+                                     dp_size=engine.engine_config.dp,
+                                     pp_size=None,
+                                     ep_size=engine.engine_config.ep,
+                                     dp_rank=engine.engine_config.dp_rank,
+                                     block_size=engine.engine_config.block_size,
+                                     num_cpu_blocks=engine.scheduler.block_manager.num_cpu_blocks,
+                                     num_gpu_blocks=engine.scheduler.block_manager.num_gpu_blocks,
+                                     available_nics=engine.engine_config.available_nics)
 
     return response.model_dump_json()
 
-@router.post("/distserve/p2p_initialize")
+
+@router.post('/distserve/p2p_initialize')
 async def p2p_initialize(init_request: DistServeInitRequest):
     return VariableInterface.async_engine.p2p_initialize(init_request)
 
-@router.post("/distserve/p2p_connect")
+
+@router.post('/distserve/p2p_connect')
 async def p2p_connect(conn_request: List[DistServeConnectionRequest]):
     return VariableInterface.async_engine.p2p_connect(conn_request)
 
-@router.post("/distserve/set_engine_to_prefill")
+
+@router.post('/distserve/set_engine_to_prefill')
 async def set_engine_to_prefill() -> JSONResponse:
```

Comment:
better combine three apis `/distserve/set_engine_to_prefill`, `/distserve/set_engine_to_decode`, `/distserve/set_engine_to_hybrid` into one as `/distserve/set_engine_role`

Reply:
it seems that these apis are not used, should we keep them?